app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -110,7 +111,7 @@ class EditProfileFragment : Fragment() {
             .observe(this, Observer {
                 Snackbar.make(rootView.editProfileCoordinatorLayout, it, Snackbar.LENGTH_LONG).show()
                 if (it == USER_UPDATED) {
-                    activity?.onBackPressed()
+                    (activity as? MainActivity)?.onSuperBackPressed()
```

Comment:
Do not cast.

Reply:
How do I use this method from MainActivity then without casting?

Reply:
Use smart cast instead.

Reply:
Changes made